frontends/web/src/components/transactions/transaction.tsx

```diff
@@ -107,6 +107,7 @@ class Transaction extends Component<Props, State> {
                 note: this.state.newNote,
             });
         }
+        this.focusEdit();
         this.setState(
             ({ editMode }) => ({ editMode: !editMode }),
             this.focusEdit,
```

Comment:
what does this do actually?

Reply:
tries to focus the input element, there is 2 issues (also see commit message) if the text is really long or sometimes the this.input isnt set or rerendered. But it still seems to not be able to focus when you reopen the same tx and edit again :(